tor_ocr/main.py

```diff
@@ -41,29 +41,37 @@
     u_tor_post_id.reply(ocr_magic)
 """
 
+# "helloworld" is a valid API key, however use it sparingly
+__OCR_API_KEY__ = os.getenv('OCR_API_KEY', 'helloworld')
 
-def process_image(local_file):
-    with PyTessBaseAPI() as api:
-        api.SetImageFile(local_file)
-        text = api.GetUTF8Text()
 
-        confidences = api.AllWordConfidences()
-        if not confidences or len(confidences) == 0:
-            # we have an image, but it *really* couldn't find anything, not
-            # even false positives.
-            return None
+def process_image(image_url):
+    """
+    Processes an image with OCR, using ocr.space
+    :param image_url: a string url of what you need OCRed
+    :return: A dictionary containing several values, most importantly 'text'
+    """
+    json_result = json.loads(ocr_space_url(image_url))
```

Comment:
I just noticed this part. It looks like we're calling `json.loads()` on JSON that has already been parsed into an object. `response.json()` is a shorter version of `json.loads(response.text)`, so there is no need to parse it again.

Reply:
I must have left that in when changing to response.json